src/util.js

```diff
@@ -0,0 +1,26 @@
+const addTask = () => {
+  class Task {
+    constructor(description) {
+      this.description = description;
+      this.index = new Date();
+      this.completed = false;
+    }
```

Comment:
- [ ] It would be better if the index of the tasks are set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) as one of the project's requirements. Currently, the index is set to the current time the task is created. Please fix this minor issue. 😄 💪 

![Capture](https://user-images.githubusercontent.com/64173189/156841014-3114d8ca-0162-490e-95ef-7700e5c54902.JPG)

Reply:
This issue is well resolved! ✔️